charts/k8ssandra/templates/stargate/service_monitor.yaml

```diff
@@ -4,13 +4,15 @@ kind: ServiceMonitor
 metadata:
   name: {{ .Release.Name }}-prometheus-{{ include "k8ssandra.datacenterName" . }}-stargate
   labels:
+    release: {{ .Release.Name }}
+    app: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate
 {{ include "k8ssandra.labels" . | indent 4 }}
 spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate
```

Comment:
With the chart refactoring we will hopefully be able to standardize on the label selectors. In #280 I have this:

```yaml
  selector:
    matchLabels:
      {{- include "k8ssandra-common.selectorLabels" . | nindent 6 }}
```